.gitignore

```diff
@@ -2,3 +2,7 @@
 *.jl.*.cov
 *.jl.mem
 deps/deps.jl
+notebooks
+docs/build
+docs/src
```

Comment:
Maybe not the best idea. You probably have these stuff locally in your computer and don't want to see those in your `git status`. But you actually can add these in your `.git/info/exclude` to avoid them showing in you `git status`.

Reply:
docs/build is probably a good idea

Reply:
Maybe file `docs/readme_header.txt` should be in the `docs/src/` folder?